.github/workflows/tests.yml

```diff
@@ -603,6 +603,8 @@ jobs:
         echo "stats" | nc -q 1 localhost 11211
     - name: Verify
       if: ${{ needs.skip_check_spark.outputs.should_skip != 'true' }}
+      env:
+          MEMCACHED: localhost:11211
```

Comment:
This is also required for the Python tests (line 222). Maybe it is better to set the environment variable globally instead of for each job (cf. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env).

Reply:
Fixed.